integration/terraform/ec2/win/variables.tf

```diff
@@ -5,22 +5,22 @@ variable "region" {
 
 variable "ec2_instance_type" {
   type    = string
-  default = "t3a.xlarge"
+  default = "g4dn.xlarge"
 }
 
 variable "ami" {
   type    = string
-  default = "cloudwatch-agent-integration-test-win-2022*"
+  default = "Windows_Server-2019-English-Deep-Learning*"
```

Comment:
Do we need to use this AMI? The integration test AMI already fixes OpenSSH

Reply:
Yes. Since we have a separate pipeline for it https://github.com/aws/amazon-cloudwatch-agent/blob/master/.github/workflows/integrationTest.yml#L457-L511